changelogs/unreleased/6269-sunjayBhatia-deprecation.md

```diff
@@ -0,0 +1,4 @@
+## Contour sample YAML manifests no longer use `prometheus.io/` annotations
```

Comment:
The provisioner also sets these on the contour & envoy workloads, do we want to drop those as well? The user can always specify them themselves if needed via ContourDeployment.

Reply:
aha yeah i forgot about those as well, I'll make some changes to address the provisioner

Reply:
looks like the provisioner already adds the envoy metrics port so no change needed there